### DISCUSSION TITLE
Is there a way to open meida extended view from other plugins?

### DISCUSSION THREAD

**Discussion by aoout** [aidenlx/media-extended#333] [Q&A] [answered]
I'm writing a plug-in and want to be able to open the corresponding media extended view via a local path. Is this possible?

**Comment by aidenlx** [comment, marked as answer]
using leaf.setViewState() with view type "mx-url-video" and state being `{url: "file:///path/to/media"}` should work.

**Reply by aoout** [reply]
After my testing, this should be `source` instead of `url`.
